js/server/modules/@arangodb/foxx/queues/manager.js

```diff
@@ -226,11 +226,6 @@ exports.manage = function () {
   }
 
   if (global.ArangoServerState.getFoxxmasterQueueupdate()) {
-    if (!isCluster) {
-      // On a Foxxmaster change FoxxmasterQueueupdate is set to true
-      // we use this to signify a Leader change to this server
-      foxxManager.healAll(true);
-    }
```

Comment:
What is the rationale for removing this code here?
If that code is going to be removed, where is the healAll being executed for single servers instead?

Reply:
this route is invoked through the foxx queues, which don't have to be enabled. 
Otoh, the `true` ends up to only unpacking foxx services on the failover host, and not registering their routes.

Reply:
What about single server here? Is there any `healAll(true)` being executed for single servers now?

Reply:
this was added by Simon especially for active failover according to git blame.

Reply:
That's fine. But apparently the `if` block was executed in single server as well.
So just wondering if it was needed on single server or not.

Reply:
The `true` does less than the default, it opts out of reloading the routing, meaning it doesn't directly take effect for the installed services (this is intended to be used in situations where doing so would result in unnecessary churn and the reloading is triggered explicitly later).

This is useless in a single server as the self-heal is only relevant if the server needs to restore services from the database, which wouldn't happen outside failover for single server deployments. One could argue whether it should happen in the startup though.

Reply:
If we wipe out a single server failover network and then reboot it, we probably want an initial selfHeal too? So we might have to add the new `healAll(false)` (or `healAll()`, which is identical) in `server/server.js` too.

Reply:
@pluma do you mean to cope with manual replication? can you push that?

Reply:
tested whether a cold restart with a new leader would also properly restore, as discussed.